backend/api/src/image_ops.rs

```diff
@@ -63,7 +63,7 @@ pub fn regenerate_images(
         let new_image = match kind {
             ImageKind::Canvas => original.resize_exact(width, height, FilterType::Nearest),
 
-            ImageKind::Sticker if (width <= original.width() && height <= original.height()) => {
+            ImageKind::Sticker if (width >= original.width() && height >= original.height()) => {
```

Comment:
turns out none of the images LARGER than the required size were being processed, while the ones smaller than were :)

Reply:
@rrcwang Shouldn't it be `||` ? i.e. if _any_ side is larger than the target?

Reply:
Good question. This is correct and (round-aboutly) implemented as the logical converse of the requirement in #1389

> For Stickers, the logic should be that if either dimension is bigger than the box, it fits the box such that neither dimension exceeds it and the largest dimension matches it.
>
> In other words, a large widescreen image will be downsized such that the width is 1440 and the aspect ratio is maintained, while a large profile image will be downsized such that the height is 810 and the aspect ratio is maintained

Here we check the condition for "not modifying the image", otherwise the `Sticker` which DOES need to be:
```
if original image fits into required dimensions:
    do not resize
else: // at least one of image dims is larger than the required dimensions
    resize image 
```

Reply:
ah, I was (mostly) just reading the lines here out of context - thought "original" meant the defined target size. My mistake!